micro-ui/web/packages/modules/workbench/src/configs/UICustomizations.js

```
@@ -1009,5 +1010,116 @@ export const UICustomizations = {
+          return row?.hierarchyType;
+          break;
+        case "LEVELS":
+          return row?.boundaryHierarchy?.length;
+          break;
+        case "CREATION_DATE":
+          let epoch = row?.auditDetails?.createdTime;
+          return Digit.DateUtils.ConvertEpochToDate(epoch);
+          // return row?.auditDetails?.createdTime;
+          break;
+        case "ACTION":
+          const tenantId = Digit?.ULBService?.getCurrentTenantId();
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)